apps/src/code-studio/pd/foorm/library_editor/FoormLibraryEditorHeader.jsx

```diff
@@ -94,20 +90,16 @@ class FoormEditorHeader extends Component {
   };
 
   render() {
+    // need to get this header to show when a "new library" is selected
     return (
       <div>
-        {this.props.formName && (
```

Comment:
we could leave both of these here, because we will have either a `formName` or a `libraryName`, so only one will be displayed